data/strings/types_strings.txt

```diff
@@ -23884,7 +23884,7 @@
     fi = Auton osat
     fr = Pièces de voiture
     hi = गाड़ी के पुर्जे
-    hu = Autó alkatrészek
+    hu = Autóalkatrészek
```

Comment:
I consider "Autóalkatrész bolt" better.

Reply:
Thanks, changed to Autóalkatrész-bolt according to [[AkH11-138](https://helyesiras.mta.hu/helyesiras/default/akh11#138), [AkH12-139](https://helyesiras.mta.hu/helyesiras/default/akh12#139), OH-128–131]